internal/api/middleware.go

```diff
@@ -33,24 +23,34 @@ func loggingMiddleware(h http.Handler) http.Handler {
 
 		// this runs handler h and captures information about HTTP request
 		m := httpsnoop.CaptureMetrics(h, w, r)
-		Log.Info(
+		slog.Info(
 			"RES",
-			slog.Int("id", reqID),
+			slog.String("request_id", reqID),
 			slog.Int("status", m.Code),
 			slog.Duration("duration", m.Duration),
 			slog.Int64("size", m.Written),
 		)
 	})
 }
 
+type userIDCtxKey struct{}
+
 func authMiddleware(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		auth := r.Header.Get("Authorization")
 		if auth == "" {
 			w.WriteHeader(http.StatusUnauthorized)
 			return
 		}
-		// TODO: validate auth
-		h.ServeHTTP(w, r) // обслуживание оригинального запроса
+		auth = auth[7:] // cut 'Bearer ' prefix
```

Comment:
Не настаиваю, но мне кажется это гораздо нагляднее.
```suggestion
		auth = strings.Replace(auth, "Bearer ", "", 1)
```